src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/VilkårsvurderingSteg.kt

```diff
@@ -60,10 +60,12 @@ class VilkårsvurderingSteg(
         beregningService.oppdaterTilkjentYtelsePåBehandling(behandling, personopplysningGrunnlag, vilkårsvurdering)
 
         // sjekker og tilpasser kompetanse skjema når vilkårer er vurdert etter EØS forordingen
-        val erNoenVilkårVurdertEtterEøsForordning = vilkårsvurdering.personResultater.any {
+        // eller det ligger allerede en kompetanse
+        val finnesKompetanserEllerVilkårErVurdertEtterEøs = vilkårsvurdering.personResultater.any {
```

Comment:
```suggestion
        val finnesKompetanserEllerVilkårVurdertEtterEøs = vilkårsvurdering.personResultater.any {
```

Småpirk, men synes finnesKompetanserEllerVilkårVurdertEtterEøs er litt mer naturlig i språket